modules/spring-config/src/main/java/org/mule/runtime/config/internal/AutoDiscoveredDependencyResolver.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.internal;
```

Comment:
move this and related cpde to its own subpackage

Reply:
all the resolvers are now in the resolvers package